int8 address # The address to write to
bool value   # The value to write to the specified address